setup.py

```diff
@@ -62,7 +62,7 @@ def get_platform():
 HOST_PLATFORM = get_platform()
 MS_WINDOWS = (HOST_PLATFORM == 'win32')
 CYGWIN = (HOST_PLATFORM == 'cygwin')
-MACOS = (HOST_PLATFORM == 'darwin')
+MACOS = (HOST_PLATFORM.startswith('darwin'))
```

Comment:
Why is this change needed? HOST_PLATFORM will be darwin on macOS.

Reply:
`HOST_PLATFORM` is set to `_PYTHON_HOST_PLATFORM` which will be `darwin-arm64` when cross compiling.

Reply:
ok.